src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_CSharp_Explicit.semantic.txt

```diff
@@ -1,8 +1,6 @@
 //line,characterPos,length,tokenType,modifier
 0 0 1 73 0 //razorTransition
 0 1 12 76 0 //razorDirective
-0 12 1 85 0 //markupTextLiteral
```

Comment:
Most of these changes reflect no longer returning something for spaces, which I wasn't super keen on in the first-place

Reply:
Baselines are dope now at least 😄